### PR TITLE
[ID] Update Do you really want to ask peppy

### PR DESCRIPTION
**Pull request by fajar13k**
- Update Bahasa Indonesia translation of `Do you really want to ask peppy` article

### CODE REVIEW

File: wiki/Do_you_really_want_to_ask_peppy/id.md [hunk 1]
```diff
@@ -1,20 +1,15 @@
----
-outdated: true
-outdated_since: d80f130d2ee1a1de26188b49560c8b6ed876fd56
----
+# Apa kamu benar-benar ingin bertanya kepada peppy?
 
-# Apakah Anda benar-benar ingin bertanya kepada peppy?
+Hai! peppy paham keinginan kamu untuk menghubunginya secara pribadi, tetapi dia meminta kamu untuk memahami terlebih dahulu bahwa ini **bukan** tempat yang kamu inginkan untuk meminta bantuan atau layanan dukungan padanya. Dia memisahkan keperluan layanan dukungan osu! dari pesan pribadi dan bahwa menjawab pesan pribadi dari forum cukup sulit untuk diikuti baginya. Karena itu ia memintamu untuk melakukan salah satu optsi berikut untuk menghubunginya:
```
Comment (vzrenggamani) [inline wiki/Do_you_really_want_to_ask_peppy/id.md:3]: typo on the last line `opsi`

File: wiki/Do_you_really_want_to_ask_peppy/id.md [hunk 2]
```diff
@@ -1,20 +1,15 @@
...
+Untuk masalah akun dan pembayaran, pelarangan tindak lanjut, pertanyaan mengenai turnamen dan **segala sesuatu yang berkaitan dengan permohonan bantuan dalam osu!**, kamu dapat mengirim pertanyaanmu ke [accounts@ppy.sh](mailto:accounts@ppy.sh) (perihal akun) atau [osu@ppy.sh](mailto:osu@ppy.sh) (pertanyaan lainnya).
 
 ## Menghubungi peppy
 
-Jika Anda ingin menghubungi peppy secara pribadi untuk kebutuhan di luar osu!, Anda dapat mengirim email kepada peppy ke alamat [pe@ppy.sh](mailto:pe@ppy.sh). Harap perhatikan bahwa permintaan perihal layanan ataupun hal yang terkait dengan osu! akan ditandai sebagai spam.
+Jika kamu ingin menghubungi peppy secara pribadi untuk kebutuhan di luar osu!, kamu dapat mengirim email kepada kepadanya di [pe@ppy.sh](mailto:pe@ppy.sh). Perlu diingat bahwa permintaan perihal layanan dukungan ataupun hal yang terkait dengan akun akan ditandai sebagai spam.
```
Comment (vzrenggamani) [inline wiki/Do_you_really_want_to_ask_peppy/id.md:15]: I am just curious about this part. Does the english translation talk about account stuff or osu! stuff?.
Its contradicting with the 1st paragraph, we could talk to peppy about stuff outside osu!, but the last paragraphy says, everything about account support marked as spam.

Can  `... yang terkait dengan akun akan ditandai sebagai spam` changed to `... yang terkait dengan akun *osu!* akan ditandai sebagai spam`.

Comment (fajar13k) [reply wiki/Do_you_really_want_to_ask_peppy/id.md:15]: Fair enough, let's apply it, since yes i didn't point out which account is directed towards spam.

Comment (vzrenggamani) [reply wiki/Do_you_really_want_to_ask_peppy/id.md:15]: Great, Gonna follow this PR for any changes..

**Pull request merged**